sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/DataFeed/DataFeedGranularity.cs

```diff
@@ -23,13 +24,14 @@ internal DataFeedGranularity(DataFeedDetail dataFeedDetail)
         }
 
         /// <summary>
+        /// The type of <see cref="DataFeedGranularity"/>.
         /// </summary>
         public DataFeedGranularityType GranularityType { get; }
 
         /// <summary>
-        /// The custom granularity value. This property must be set if <see cref="DataFeedGranularity.GranularityType"/> is <see cref="DataFeedGranularityType.Custom"/>.
+        /// The custom granularity value, in seconds. This property must be set if <see cref="GranularityType"/>
+        /// is <see cref="DataFeedGranularityType.Custom"/>.
         /// </summary>
-        /// <value></value>
         public int? CustomGranularityValue { get; set; }
```

Comment:
For later, should the concept of seconds or time be in the name?

Reply:
Added to the "design considerations" issue. I agree.